src/content/docs/2/guide/recipe/notifications.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Notifications
+description: Stay tuned while we work on some delicious recipes.
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+In this recipe we'll take a look at sending a native notification to your user. We'll do this by using the official notification plugin.
+
+## Setup
+
+First you have to install the notification plugin in your backend.
+
+<Tabs>
+    <TabItem label="Auto">
+    ```bash
+    tauri plugin add notification
+    ```
+    </TabItem>
+    <TabItem label="Manual">
+        <Tabs>
+            <TabItem label="Cargo.toml">
+            ```toml
+            [dependencies]
+            tauri-plugin-notification = "2.0.0-alpha"
+            ```
+            </TabItem>
+            <TabItem label="lib.rs">
+            ```rust
+            #[cfg_attr(mobile, tauri::mobile_entry_point)]
+            pub fn run() {
+                tauri::Builder::default()
+                    // Initialize the plugin
+                    .plugin(tauri_plugin_notification::init())
+                    .run(tauri::generate_context!())
+                    .expect("error while running tauri application");
+            }
+            ```
+            </TabItem>
+        </Tabs>
```

Comment:
UX feedback: If they are sequential steps and not mutually exclusive then don't use tabs. If it's "action A AND action B must be done" then list those out without any tabs. If it's "action B OR Action B" then use tabs (like you've done with Automatic vs. Manual here)